### DISCUSSION TITLE
How to stick a node position in different viewport sizes?

### DISCUSSION THREAD

**Discussion by hiimnhan** [xyflow/xyflow#4410] [Q&A]
For example, I have a map like this. I want to stick the Azure node to Japan. It lies a little bit far or nearer where I want on different viewport when I test on different screen (laptop screen, 27 inches screen,...). Assume that it won't have any interaction like zoom or pan. 
My question is how to stick a node to a certain position  with different screen sizes?

![image](https://github.com/xyflow/xyflow/assets/35358825/f1028ae8-1895-494b-9ff7-5232dbb3a9cc)

**Comment by moklick** [comment]
How are you getting the flow coordinate? I would assume that you have a lat/lng coordinate, that you turn into a screen coordinate with the map library and then you are using screenToFlowPosition? Is that right?

**Reply by hiimnhan** [reply]
well no. It is just a div that I set `background-image` which I put as children of `<ReactFlow />`. I assume that each node position will be set relatively with the div but seems like it is not. What I want is the node should be like pin to the point on the div no matter what size of the screen.

**Reply by moklick** [reply]
It's probably because how the background image gets positioned. Could you create a codesandbox for this: https://new.reactflow.dev ?
